src/components/character-count/character-count.test.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * @jest-environment ./lib/puppeteer/environment.js
+ */
+/* eslint-env jest */
+
+const configPaths = require('../../../config/paths.json')
+const PORT = configPaths.ports.test
+
+let browser
+let page
+let baseUrl = 'http://localhost:' + PORT
+
+beforeEach(async () => {
+  browser = global.__BROWSER__
+  page = await browser.newPage()
+})
+
+afterEach(async () => {
+  await page.close()
+})
+
+describe('Character count', () => {
+  describe('when JavaScript is unavailable or fails', () => {
+    it('shows the static message', async () => {
+      await page.setJavaScriptEnabled(false)
+
+      await page.goto(baseUrl + '/components/character-count/preview', { waitUntil: 'load' })
+      const message = await page.evaluate(() => document.body.querySelector('.govuk-character-count__message').innerHTML.trim())
```

Comment:
Non-blocking: Lines like this are very long, can we break them up?

```suggestion
      const message = await page.evaluate(() => {
        return document.body.querySelector('.govuk-character-count__message').innerHTML.trim()
      })
```

Alternatively, it looks like we might be able to use `page.$eval`? (not tested)

```suggestion
      const message = await page.$eval('.govuk-character-count__message', el => el.innerHTML.trim())
```

Reply:
Just tried `page.$eval` out in #1056 and it seems to work well for things like this 👍